Gems/Multiplayer/Code/Source/Components/NetworkTransformComponent.cpp

```diff
@@ -119,6 +121,18 @@ namespace Multiplayer
         }
     }
 
+    void NetworkTransformComponent::OnCorrection()
+    {
+        // Snap to latest
+        OnResetCountChangedEvent();
+
+        // Hard set the entities transform
+        if (!GetTransformComponent()->GetWorldTM().IsClose(m_targetTransform))
+        {
+            GetTransformComponent()->SetWorldTM(m_targetTransform);
+        }
```

Comment:
Should this be interpolating the transform using blend factor?

Reply:
Nope, this is a hard reset to server state after a correction. We need this exact so that input replay generates a correct output state that is again synchronized with the server.